documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1254,6 +1259,24 @@ However, when `hstore.handling.mode` configuration property is set to `json`, th
 
 |=======================
 
+[[domain-types]]
+==== PostgreSQL Domain Types
+
+PostgreSQL also supports the notion of user-defined types that are based upon other underlying types.
+When such column types are used, Debezium the column's representation based on the full type hierarchy.
+
+[IMPORTANT]
+====
+Special consideration should be taken when monitoring columns that use domain types.
+
+When a column is defined using a domain type that extends one of the default database types and the domain type defines a custom length/scale, the generated Schema will inherit that defined length/scale.
+
+When a column is defined using a domain type that extends another domain type that defines a custom length/scale, the generated Schema will **not** inherit the defined length/scale because the PostgreSQL driver's column metadata implementation.
```

Comment:
```suggestion
When a column is defined using a domain type that extends another domain type that defines a custom length/scale, the generated schema will **not** inherit the defined length/scale because the PostgreSQL driver's column metadata implementation.
```

Reply:
Ok, so to make sure IUC: nested domain type *are* somewhat supported, meaning the right value type will be exported, but the schema would lack the scale/width parameters, correct?

Reply:
Yes exactly, which is why I added https://github.com/debezium/debezium/pull/1079/files#diff-31a43403a1b613fa67bb5278ae301513R548 in hopes that if this limitation can be improved upon, this test not failing means we can come back and revisit it.

Reply:
Ok, thanks!